stdlib/public/Concurrency/linker-support/magic-symbols-for-install-name.c

```diff
@@ -22,116 +22,55 @@
 
 #if defined(__APPLE__) && defined(__MACH__) && SWIFT_STDLIB_SUPPORT_BACK_DEPLOYMENT
 
-#include <Availability.h>
+#include <mach-o/loader.h>
 #include <TargetConditionals.h>
 #include "swift/shims/Visibility.h"
 
-#define RPATH_INSTALL_NAME_DIRECTIVE_IMPL2(name, major, minor) \
-  SWIFT_RUNTIME_EXPORT const char install_name_ ## major ## _ ## minor \
-  __asm("$ld$install_name$os" #major "." #minor "$@rpath/lib" #name ".dylib"); \
-  const char install_name_ ## major ## _ ## minor = 0;
+// Concurrency was supported as an embedded library in macOS (née OS X) 10.9, iOS 7.0, watchOS 2.0, tvOS 9.0.
```

Comment:
Technically, Concurrency only back-deploys as an embedded library to macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0

Reply:
Ah, I figured the original code was probably wrong to back deploy all the way to the original Swift versions.